### ISSUE TITLE
`Plots` cannot add plots to same device/screen when using `quickPlot::Plot`

### ISSUE THREAD

**Issue by CeresBarros** [PredictiveEcology/SpaDES.core#170]
Because the the plotting data object is internally always called `data`, `quickPlot::Plot` will always overwrite the previous plot:
```
library(SpaDES.core)
library(quickPlot)
r <- raster::raster(extent(0,10,0,10), vals = sample(1:3, size = 100, replace = TRUE))
r2 <- raster::raster(extent(0,10,0,10), vals = sample(5:7, size = 100, replace = TRUE))
Plots(r, title = "Biomass", new = TRUE)
Plots(r2, title = "blah", new = TRUE)   ## overwrites the previous instead of rearrigning like:

clearPlot(force = TRUE)
Plot(r)
Plot(r2)
```

**Comment by CeresBarros** [comment]
PR  #169 suggests a fix

**Comment by CeresBarros** [comment]
fixed in #169, which was merged into development
